code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -100,10 +100,10 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	name = "Green Beer"
 	description = "An alcoholic beverage brewed since ancient times on Old Earth. This variety is dyed a festive green."
 	color = "#A8E61D"
-	taste_description = "green piss water"
+	taste_description = "crayons and piss"
```

Comment:
the entire joke is that some beers gets called green piss water, why rename this to include crayons? ruins the joke.

Reply:
> 
> 
> the entire joke is that some beers gets called green piss water, why rename this to include crayons? ruins the joke.

my bad man I fixed it to it's original state